packages/nextjs/src/config/loaders/ast.ts

```diff
@@ -25,16 +25,26 @@ const jscs = jscodeshiftDefault || jscodeshiftNamespace;
 
 // These are types not in the TS sense, but in the instance-of-a-Type-class sense
 const {
-  ExportSpecifier,
+  Node,
   Identifier,
-  ImportSpecifier,
+  JSXIdentifier,
   MemberExpression,
-  Node,
+  ClassDeclaration,
+  FunctionDeclaration,
   ObjectExpression,
   ObjectPattern,
+  ArrayPattern,
+  RestElement,
   Property,
-  VariableDeclaration,
   VariableDeclarator,
+  VariableDeclaration,
+  ImportSpecifier,
+  ExportSpecifier,
+  ExportNamedDeclaration,
+  ExportAllDeclaration,
+  ExportDefaultDeclaration,
+  ExportDefaultSpecifier,
+  TSTypeParameter,
```

Comment:
Nit: Can we please put these back into alphabetical order?

(Another nit, for lack of a better place to put it: Our convention is to use backticks rather than quotes for code-type things.)

Reply:
Sure: ad46cdb1739e69da2fe96761cc5d2ea2d47d08ac

As for the quotes and stuff I'll try to go through the comments and clean it up a bit. I generally agree, it's only a little bit messy cause I was rushing stuff.